CODE_OF_CONDUCT.md

```diff
@@ -57,7 +57,9 @@ reported by contacting select project maintainers, currently azul@autocrypt.org
 and compl4xx@autocrypt.org -- they are also often on the IRC #autocrypt channel
 on freenode. They will review reports and involve other maintainers as appropriate,
 resulting in a response that is deemed neccessary and appropriate to the circumstances.
-Maintainers are obligated to maintain confidentiality with regard to the reporter of an incident.
+Maintainers are obligated to maintain confidentiality with regard to the reporter
+of an incident. They should as well believe a reporter, and respect their experience
+and perspective.
```

Comment:
To be honest, i find "the should believe" an odd formulation here.  IMO especially the "enforcement" folks are to adhere to the "good" behaviour we list above, which includes empathy, respect etc.

Reply:
The belief of a victim on part of being a victim is a redundancy.
The belief of a reporter in being the reporter is guilt by condemnation.

There was no ," in respecting". Logic ensues.